scripts/local_envs/set_my_env_vars.bash

```diff
@@ -0,0 +1,6 @@
+cd "$(dirname "${BASH_SOURCE[0]}")"
```

Comment:
Can we actually not push scripts folder to the public repo if that's okay. I like all the other changes, but some of the scripts assume, for example, you have sudo and use bashrc, which isn't always true. They are more local scripts.

Reply:
removed